soc/arm/st_stm32/stm32f4/dts.fixup

```diff
@@ -80,6 +80,36 @@
 #define CONFIG_SPI_6_NAME			ST_STM32_SPI_40015400_LABEL
 #define CONFIG_SPI_6_IRQ			ST_STM32_SPI_40015400_IRQ_0
 
+#define CONFIG_I2S_1_BASE_ADDRESS               ST_STM32_I2S_40013000_BASE_ADDRESS
```

Comment:
Nitpick: Can you align all these ?
Seems #1 and #2 are one or 2 chars away from others

Reply:
This file is using sometimes tabs and sometimes spaces. At line 83 I started with spaces. I'll make sure to always use tabs.

Reply:
@avisconti , cf #10416, moving all stm32 fixup files to use spaces.

Reply:
I see... OK, I'll use always spaces.